corehq/apps/auditcare/models.py

```diff
@@ -171,22 +183,19 @@ def audit_view(cls, request, user, view_func, view_kwargs, extra={}):
 )
 
 
+@foreign_value_init
 class AccessAudit(AuditEvent):
-    access_type = StringProperty(choices=ACCESS_CHOICES)
-    ip_address = StringProperty()
-    session_key = StringProperty()  # the django auth session key
-
-    user_agent = StringProperty()
-
-    get_data = StringListProperty()
-    post_data = StringListProperty()
-    http_accept = StringProperty()
-    path_info = StringProperty()
-
-    failures_since_start = IntegerProperty()
-
-    class Meta(object):
-        app_label = 'auditcare'
+    access_type = models.CharField(max_length=16, choices=ACCESS_CHOICES)
+    ip_address = models.CharField(max_length=45, blank=True, default='')
```

Comment:
I think it's mainly for legacy reasons. I'm also not familiar with `GenericIPAddressField`, so don't know the constraints. Is that backed by a different data type? Not sure if the historical data we have in this field in Couch would fit into it? Hopefully it would, but you never know until you've looked at the data.

I'm not sure it matters that much, to be honest. Is this a blocker for you?

Reply:
Eck...it's not a blocker, but _hqwebapp/models:UserAccessLog_ is using `GenericIPAddressField` and I'd like to be consistent if possible. Pulled up the migration code for that, and it doesn't look like its backed by a string value: `"ip" inet NULL` is the field definition. Should we just overlook the discrepancy? I'm not sure changing `UserAccessLog` would be that easy either